engine/src/mbliphoneapp.mm

```diff
@@ -360,8 +364,18 @@ - (BOOL)application:(UIApplication *)p_application didFinishLaunchingWithOptions
     NSDictionary *t_push_notification = [p_launchOptions objectForKey: UIApplicationLaunchOptionsRemoteNotificationKey];
     // MM-2012-09-25: [[ Bug 10391 ]] Retain notification.
     if (t_push_notification)
-        m_pending_push_notification = [[t_push_notification objectForKey:@"payload"] retain];
-
+	{
+		// Prevent crash when launching the app from the notifications screen:
+		// If the payload is empty, JSON null will be deserialized to NSNull. Unlike nil,
+		// we cannot send (most) messages to NSNull
+		id t_pending_push_notification_value = [[t_push_notification objectForKey:@"payload"] retain];
```

Comment:
Ah sorry - I see - don't retain here - retain when you assign it to m_pending_push_notification :)